### ISSUE TITLE
swift: Timeout sometimes uploading to Rackspace Cloudfiles

### ISSUE THREAD

**Issue by rphillips** [ncw/rclone#54]
I am running about 25 builders on buildbot, and on every build about 2-5 builds fail to upload to cloudfiles with the following timeout:

```
2015/05/07 03:15:43 dists/cloudmonitoring/main/binary-i386/Packages.gz: Failed to copy: Timeout when reading or writing data
```

Usually a subsequent build works fine.

**Comment by ncw** [comment]
This isn't the only complaint I've had about rclone and cloudfiles.  I think the default timeouts may be too short (10 seconds connect, 60 seconds data) and they aren't configurable unfortunately.

I'll fix that in a new release.

**Comment by rphillips** [comment]
Appreciate it. Maybe even a ```--retries=N``` to retry a file N times before failing

**Comment by zeshanb** [comment]
--bwlimit=0: Zero is unlimited. Might like to set this to a sweet spot.
Bandwidth limit in kBytes/s, or use suffix K|M|G

-transfers=1: Number of file transfers to run in parallel.

--checkers=2: Number of checkers to run in parallel.

@Ryan Have you tried these options when running the rClone command? when I
decreased the number of checkers and parallel transfers. Transfer was able
to continue for longer period. I've noticed with number of parallel
transfers and checkers can place a lot of load on the rClone system
process.

**Comment by rphillips** [comment]
I will give that a shot. Thanks.

**Comment by shackra** [comment]
I experience this bug too

**Comment by ncw** [comment]
I've released v1.13 which introduces the `--timeout` and `--conntimeout` flags which have larger defaults than the swift module used to use.

If you still see problems, then please experiment with the values using these flags and I can change the defaults.

Thanks for the bug report

-- Nick

**Comment by ramihussain** [comment]
Hi,
I am experiencing timeouts failed to copy, using rclne version v1.17.

I have set the flag --timeout=0 and deafult --conntimeout.  Any hints will be appreciated.

Failed to copy: Timeout when reading or writing data
